src/Umbraco.Web.UI.Client/src/installer/steps/user.html

```diff
@@ -1,7 +1,7 @@
 ﻿<div ng-controller="Umbraco.Install.UserController">
-    <h1>Install Umbraco 8</h1>
+    <h1>Install Umbraco 9</h1>
```

Comment:
Now you've made it official :P

Reply:
🙈 Not more then the assembly version. But actual I guess we should use the assembly version instead of a hardcoded value.